ledger-tool/src/main.rs

```diff
@@ -2331,6 +2331,12 @@ fn main() {
                         .required(false)
                         .help("Number of slots in the output"),
                 )
+                .arg(
+                    Arg::with_name("exclude_vote_only_slots")
```

Comment:
Given where the conversation was trending with wanting to have a separate tool broken out that is more user friendly for operators, I changed the code back to show "all slots" by default, and have user opt in to excluding vote only slots.

The rationale for this was to retain "general purpose" (excluding vote only slots is inherently kind of restart minded), but I could still see the desire for excluding vote only by default. I'm a little on the fence about this so curious if you two have strong thoughts one way or another.